app/presenters/tree_builder_ops_rbac_features.rb

```diff
@@ -0,0 +1,160 @@
+class TreeBuilderOpsRbacFeatures < TreeBuilder
+  has_kids_for Menu::Section,     [:x_get_tree_section_kids]
+  has_kids_for Menu::Item,        [:x_get_tree_item_kids]
+  has_kids_for MiqProductFeature, [:x_get_tree_feature_kids]
+
+  attr_reader :role, :features, :editable, :root_counter
```

Comment:
If it's the end of the world, I'll change it, but I'll let others [argue](https://github.com/bbatsov/ruby-style-guide/issues/538#issuecomment-177751337) [why](https://github.com/bbatsov/ruby-style-guide/issues/538#issuecomment-178006875).